docs/products/opensearch/concepts/access_control.rst

```diff
@@ -0,0 +1,81 @@
+Access control
+==============
+
+Aiven for OpenSearch supports index-level access control lists (ACL) to control permissions.
```

Comment:
Can we say a little bit about why this is useful? For example, to grant access to create or change data, restrict access to certain data sets for reading, something else? Probably the word "security" belongs here too.

Reply:
Done ;)